source/Mlos.Python/mlos/OptimizerEvaluationTools/OptimumOverTime.py

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+import pandas as pd
+
+from mlos.Optimizers.OptimumDefinition import OptimumDefinition
+from mlos.Optimizers.OptimizationProblem import OptimizationProblem
+from mlos.Spaces import Point
+
+class OptimumOverTime:
+    """Keeps track of an optimum over time.
+

```

Comment:
Expand the comment to specify that it keeps track of a single optimum definition over time.